content/projects/katas/level-1/_index.md

```diff
@@ -57,6 +57,15 @@ In other words, there should be no directories. Only modules.
 
 If you are writing JS code then your files would follow a similar pattern. You would just have `.js` instead of `.py` files.
 
+If you are using Java, your files should be named like this:
+
+```
+├── Task1.py
```

Comment:
These are py files

Reply:
Should the learners be using Gradle at this point? I'm not sure, just double-check where gradle comes into the syllabus if you are not sure

Reply:
I believe they should not be using it at this point.